typescript/cli/tests/warp.zs-test.ts

```diff
@@ -0,0 +1,138 @@
+import { JsonRpcProvider } from '@ethersproject/providers';
```

Comment:
nbd but you can avoid these inconsistent import paths from `ethers` by reaching into the export namespaces like `import {providers}... providers.JsonRpcProvider`